src/monexamine.cpp

```diff
@@ -688,8 +685,7 @@ bool monexamine::pet_menu( monster &z )
 
     amenu.text = string_format( _( "What to do with your %s?" ), pet_name );
     if( z.has_flag( mon_flag_EATS ) ) {
-        amenu.text = string_format( _( "What to do with your %s?\n" "Fullness: %i%%" ), pet_name,
```

Comment:
I realized after patching that this is actually a string change, that we'd prefer to avoid.

We could just add a dummy `100` value to replace the function call, keeping the string unchanged in 0.I. I'm not sure that's a *good* idea, it would suggest that there is "fullness" in 0.I, but we could do it.